### ISSUE TITLE
[FEATURE]would you provide the source files used in paper 1.0

### ISSUE THREAD

**Issue by smacaijicoder** [MouseLand/cellpose#822]
https://github.com/MouseLand/cellpose/blob/579299731cb815e43d1d07bcf89668972e185292/paper/1.0/performance_figs.py#L31-L37

I am interested in the  ap values of different models, however there are only curves but no specific values in Fig 4.d-g. Could you please provide the file of mask.npy mentioned in these lines of code? Thank you very much

**Comment by smacaijicoder** [comment]
Sorry to trouble you, I also have some questions about the result of AP50=0.77 in the paper. What is the size of the image when calculate the ap？Original img size or you resize them to a fixed value like 224×224 to fit the model. If my understanding is correct, the size of the image input in the training is 224, so the model output size is also 224 right? What did you do when the size of the image in the test dataset is not 224. Looking forward to your reply.

**Comment by carsen-stringer** [comment]
We used the test images at their original sizes, and estimated the diameters. You can see the function we ran here (with the four cellpose networks): https://github.com/MouseLand/cellpose/blob/main/paper/1.0/cp_unets.py. The `Cellpose` model runs both the size estimation and the mask estimation, so the current way to run the benchmark would be:
```
from cellpose import models, io, metrics

img_test = [io.imread(root / "test" / f"{i:03d}_img.tif")
                 for i in range(68)]
masks_test = [io.imread(root / "test" / f"{i:03d}_masks.tif")
                 for i in range(68)]
seg_model = models.Cellpose(gpu=True, model_type="cyto", net_avg=True)
masks = seg_model.eval(img_test, diameter=None, channels=[1,2], channel_axis=0, normalize=True)[0]
ap, tp, fp, fn = metrics.average_precision(masks_test, masks)
print(ap.mean(axis=0))

```

**Comment by smacaijicoder** [comment]
@carsen-stringer Hello, thank you for your reply before. Recently, I encountered some problems when replicating the results of the paper. I hope to get your help.
I follow the tutorials in https://cellpose.readthedocs.io/en/latest/train.html to train the cyto model from scratch
`python -m cellpose --train --train_size --use_gpu --dir ../dataset/train/ --test_dir ../dataset/test/ --img_filter _img --pretrained_model None --chan 2 --chan2 1 --verbose`
and then use `python -m cellpose --dir ../dataset/test/ --pretrained_model ../dataset/train/models/cellpose_residual_on_style_on_concatenation_off_train_2023_11_29_19_37_26.221582 --save_png --verbose --chan 2 --chan2 1 --diameter 0 --use_gpu` to get the output mask.
the code to  get the AP as follows:
```
thresholds = np.arange(0.5, 1.05, 0.05) 
test_root = '../dataset/test/'
ntest = len(glob(os.path.join(test_root, '*_img.png')))
test_labels = [imread(os.path.join(test_root, '%03d_masks.png'%i)) for i in range(ntest)]
masks = [imread(os.path.join(test_root, '%03d_img_cp_masks.png'%i)) for i in range(ntest)]
aps=metrics.average_precision(test_labels, masks,threshold=thresholds)[0]
map = aps.mean(axis=0)
print(map)
```
However in this way the AP is [0.69197065 0.67378473 0.648926   0.6193165  0.5745736  0.50456834 0.40362763 0.2867241  0.14155634 0.02795627 0.        ]
Then I use `python -m cellpose --dir ../dataset/test/ --pretrained_model cyto --save_png --verbose --chan 2 --chan2 1 --diameter 0 --use_gpu` and the same pipeline as above, the result is [0.7952877  0.7735594  0.7493472  0.71238947 0.6623948  0.5847183 0.46891114 0.33228996 0.1629274  0.02774804 0.        ]

Why is there such a big difference between using the cyto model and my own train from scratch model?

By the way, I also tried to train with the pre-trained model cyto and train it again on the cellpose dataset, `python -m cellpose --train --use_gpu --dir ../dataset/train/ --test_dir ../dataset/test/ --img_filter _img --pretrained_model cyto --chan 2 --chan2 1 --verbose --batch_size 32 --model_name_out pre-cyto`
The result is [0.7194467  0.69905585 0.6710281  0.63346004 0.586424   0.5054282 0.3956902  0.27338412 0.13454683 0.02399613 0.        ], still lower than cyto model and paper.

It is worth mentioning that the results AP obtained by the two test methods are different, despite both using the cyto model
One way as above is [0.7952877  0.7735594  0.7493472  0.71238947 0.6623948  0.5847183 0.46891114 0.33228996 0.1629274  0.02774804 0.        ]
Another way as you reply before like:
```
root = '../dataset/test/'
img_test = [io.imread(os.path.join(root, '%03d_img.png'%i)) for i in range(68)]
masks_test = [io.imread(os.path.join(root, '%03d_masks.png'%i)) for i in range(68)]
model = models.Cellpose(gpu=True, model_type="cyto", net_avg=True)
masks = model.eval(img_test, diameter=0, channels=[2,1],normalize=True)[0]
ap, tp, fp, fn = metrics.average_precision(masks_test, masks)
print(ap.mean(axis=0))
```
this way the result is [0.7859223  0.5668159  0.14850779] (Here are only three values because the thresholds is not changed）

I'm really confused about these differences in results and hope to get your help.
